GameOfRunes/GameOfRunes/Models/Stage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Stage.swift
+//  GameOfRunes
+//
+//  Created by Jermy on 29/3/20.
+//  Copyright © 2020 TeamHoWan. All rights reserved.
+//
+
+import UIKit
+
+struct Stage {
+    enum Category: String {
+        case normal
+        case boss
+    }
+    enum AchievementLevel: String {
+        case S
+        case A
+        case B
+        case C
+        case empty
+    }
+    
+    let name: String
+    let groupName: String
```

Comment:
what is groupName used for?

Reply:
Oh right, I should rename `groupName` to `chapter`.